services/collaboration/pkg/cs3wopiserver/start.go

```diff
@@ -0,0 +1,43 @@
+package cs3wopiserver
```

Comment:
maybe a different package name? the whole service was renamed from wopi to collaboration - mayb this package deserves a rename as well .....

Reply:
I think the package itself will likely be removed when we refactor the code of the service. I'd expect the code to be moved elsewhere, so I'm not sure if it's worthy to rename the package now.